clang/docs/ReleaseNotes.rst

```diff
@@ -586,6 +586,20 @@ Improvements to Clang's time-trace
 - Clang now specifies that using ``auto`` in a lambda parameter is a C++14 extension when
   appropriate. (`#46059: <https://github.com/llvm/llvm-project/issues/46059>`_).
 
+Improvements to Coverage mapping
+--------------------------------
+
+- Macros defined in system headers are not expanded in coverage
+  mapping. Conditional expressions in system header macros are no
+  longer taken into account for branch coverage. They can be included
+  with ``-mllvm -system-headers-coverage``. (#78920)
```

Comment:
I don't think the PR/issue number will get linked automatically, so it's better to do it explicitly (see e.g. 46059 above).